projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/ConvertJobToNotebook.tsx

```diff
@@ -60,6 +71,9 @@ export async function showConvertJobToNotebookDialog() {
     if (confirmation.button.accept) {
       let { message, status } = await jobConvertToNotebookRequest();
```

Comment:
use const instead of let.

Reply:
@murphp15 i will be fixing those, i got some warnings

Reply:
Actually i realised we redefine message, so we cannot make it const